### ISSUE TITLE
Document X-Forwarded-For

### ISSUE THREAD

**Issue by redox** [algolia/algoliasearch-client-readme-generator#32]

**Comment by CBaptiste** [comment]
Hello @redox, I am not sure I understand what is missing ? I tried to look on other readme's but no success !

**Comment by redox** [comment]
Yes, this is something we haven't documented in the README so far. We have a few mentions in the [REST API documentation](https://www.algolia.com/doc/rest).

The idea is to document how you can specify an end-user IP address on our API clients to benefit from it while using `aroundLatLngViaIP` and the search (for analytics).
